fhir-bulkimportexport-webapp/src/main/java/com/ibm/fhir/bulktools/Main.java

```diff
@@ -154,6 +194,14 @@ public void parseArgs(String[] args) {
                     throw new IllegalArgumentException("Missing value for --numberOfFiles argument at posn: " + i);
                 }
                 break;
+            case "--numberOfTotalResources":
```

Comment:
best to leave these parameters as `lower` case and not mixed case

Reply:
make sense!

Reply:
done